src/components/LocationSearch/index.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { useIntl } from 'react-intl'
+import { ReactComponent as GreenBack } from '../../assets/icons/green-back.svg'
+import { ReactComponent as MagnifyingGlass } from '../../assets/icons/magnifying-glass.svg'
+import { ReactComponent as ResetSearch } from '../../assets/icons/reset-search.svg'
+export default function LocationSearch({
+  searchValue,
+  setSearchValue,
+  className,
+  focused,
+  setFocus,
+}) {
+  const { formatMessage } = useIntl()
+  const placeholder = formatMessage({
+    id: 'locationSearch:Search',
+    defaultMessage: 'Search ...',
+  })
+  function getIcon() {
+    if (focused)
+      return (
+        <IconBtn
+          type="button"
+          onClick={() => {
+            setFocus(false)
+          }}
+        >
+          <GreenBack className="left-icon" />
+        </IconBtn>
+      )
+    return <MagnifyingGlass className="left-icon" />
+  }
+
+  function getClearBtn() {
+    if (searchValue !== '')
```

Comment:
This condition can be if (searchValue === '') return '';
without wraping block of code;